hello_world/java/math_operations.java

```diff
@@ -0,0 +1,31 @@
+public class math_operations {
```

Comment:
create a directory named mathematical_operations directory and move the file there.